docs/how-to-guides/enable-streaming-mode.md

```diff
@@ -227,7 +227,7 @@ If the response code is "424 Model Error", it means that the error is caused by
 
 ### Consume using Python
 
-We have created [a utility file](../media/how-to-guides/how-to-enable-streaming-mode/scripts/event_stream.py) as an example to demonstrate how to consume the server-sent event. A sample usage would like:
+A sample usage would like:
```

Comment:
We might need to say from where to import the SSEClient class and where to install the pypi package.

Reply:
Thanks for the reminder, Zhidong! I've added the import and installation instructions to the snippet.